### ISSUE TITLE
Unable to use profiler

### ISSUE THREAD

**Issue by Mushoz** [ROCmSoftwarePlatform/tensorflow-upstream#1945]
### Issue Type

Bug

### Source

source

### Tensorflow Version

2.11.0

### Custom Code

Yes

### OS Platform and Distribution

Archlinux kernel: 6.0.12

### Python version

3.9

### CUDA/cuDNN version

ROCM 5.4.0

### GPU model and memory

6900XT

### Current Behaviour?

I am unable to run the profiler without tensorflow-rocm crashing.

### Standalone code to reproduce the issue

This example triggers the issue:

```
import tensorflow as tf
import tensorflow as np

tboard_callback = tf.keras.callbacks.TensorBoard(log_dir = '../logs/',
                                                 histogram_freq = 1,
                                                 profile_batch = '500,520')

features = np.random.randn(10000,25)
targets = np.random.randn(10000)

model = tf.keras.Sequential([
     tf.keras.layers.Dense(1)
])

model.compile(optimizer=tf.keras.optimizers.Adam(learning_rate=1e-3),
              loss=tf.keras.losses.MeanSquaredError())

model.fit(x=features, y=targets, callbacks=[tboard_callback])
```

### Relevant log output

```
Fatal Python error: Aborted


Main thread:
Current thread 0x00007f3889bec740 (most recent call first):
  File "/home/jaap/.local/share/virtualenvs/code-NonYUw1A/lib/python3.10/site-packages/tensorflow/python/profiler/profiler_v2.py", line 117 in start
  File "/home/jaap/.local/share/virtualenvs/code-NonYUw1A/lib/python3.10/site-packages/keras/callbacks.py", line 2882 in _start_profiler
  File "/home/jaap/.local/share/virtualenvs/code-NonYUw1A/lib/python3.10/site-packages/keras/callbacks.py", line 2672 in _init_profile_batch
  File "/home/jaap/.local/share/virtualenvs/code-NonYUw1A/lib/python3.10/site-packages/keras/callbacks.py", line 2421 in __init__
  File "/home/jaap/Dropbox/Projects/Google_Trends_Analysis/code/testNN.py", line 12 in <module>
  File "/home/jaap/.local/share/virtualenvs/code-NonYUw1A/lib/python3.10/site-packages/spyder_kernels/py3compat.py", line 356 in compat_exec
  File "/home/jaap/.local/share/virtualenvs/code-NonYUw1A/lib/python3.10/site-packages/spyder_kernels/customize/spydercustomize.py", line 469 in exec_code
  File "/home/jaap/.local/share/virtualenvs/code-NonYUw1A/lib/python3.10/site-packages/spyder_kernels/customize/spydercustomize.py", line 611 in _exec_file
  File "/home/jaap/.local/share/virtualenvs/code-NonYUw1A/lib/python3.10/site-packages/spyder_kernels/customize/spydercustomize.py", line 524 in runfile
  File "/tmp/ipykernel_99678/3578018583.py", line 1 in <cell line: 1>


Restarting kernel...
```

**Comment by schung-amd** [comment]
Hi @Mushoz, sorry for the long delay. I can see this issue is fairly old; are you still experiencing this/is this still relevant to you?

**Comment by schung-amd** [comment]
I can't reproduce this on 7900XTX + Ubuntu 24.04 + ROCm 6.2.4 + TensorFlow 2.16.2 (from stable Python 3.10 wheel in https://repo.radeon.com/rocm/manylinux/, set `ROCM_PATH=/opt/rocm` to address known issue), with `import tensorflow as np` corrected to `import numpy as np` and numpy 1.24.4 (using `tf.random.normal` also works).

Closing this for now as it seems to be stale, but feel free to comment if you are still experiencing this with current software versions and we can reopen to investigate.
